docs/Changelog.md

```diff
@@ -4,6 +4,112 @@ All notable changes to this project will be documented in this file.
+* Implement analyzer and codefix to switch to Assert.ThrowsExactly\[Async\] by @Youssef1313 in [#4459](https://github.com/microsoft/testfx/pull/4459)
+* Update ExpectedException analyzer message/description to use the new Assert.Throws methods (and fix typo) by @Youssef1313 in [#4527](https://github.com/microsoft/testfx/pull/4527)
+* Add ability to ignore specific test cases by @Youssef1313 in [#4457](https://github.com/microsoft/testfx/pull/4457)
+* Implement codefix for MSTEST0026: Avoid conditional access in assertions (and small analyzer false positive) by @Youssef1313 in [#4502](https://github.com/microsoft/testfx/pull/4502)
+* Add messageBuilder overloads for Assert.Throws\[Exactly\]\[Async\] by @Youssef1313 in [#4541](https://github.com/microsoft/testfx/pull/4541)
+* Implement codefix for 'Are\[Not\]Same' usage with value types by @Youssef1313 in [#4584](https://github.com/microsoft/testfx/pull/4584)
+* Reword unhandled TestMethodAttribute.Execute exception message by @Evangelink in [#4641](https://github.com/microsoft/testfx/pull/4641)
+* Implement analyzer to warn for use of Assert methods in async void by @Youssef1313 in [#4640](https://github.com/microsoft/testfx/pull/4640)
+* Introduce `RetryAttribute` for test methods by @Youssef1313 in [#4586](https://github.com/microsoft/testfx/pull/4586)
+* Fix bad nullability annotation on CultureInfo by @Evangelink in [#4670](https://github.com/microsoft/testfx/pull/4670)
+* Obsolete InternalTestFailureException and GenericParameterHelper by @Evangelink in [#4703](https://github.com/microsoft/testfx/pull/4703)
+* Remove incorrect isReportedAtCompilationEnd for UseParallelizeAttributeAnalyzer by @Youssef1313 in [#4737](https://github.com/microsoft/testfx/pull/4737)
+* Implement MSTestAnalysisMode by @Youssef1313 in [#4712](https://github.com/microsoft/testfx/pull/4712)
+* Add Assert.HasCount by @Evangelink in [#4748](https://github.com/microsoft/testfx/pull/4748)
+* Add UsingMSTestSdk property by @aelij in [#4754](https://github.com/microsoft/testfx/pull/4754)
+* Add Assert.HasCount, Assert.IsEmpty, Assert.IsNotEmpty and Assert.ContainsSingle by @Evangelink in [#4756](https://github.com/microsoft/testfx/pull/4756)
+* Support conditional tests by @Youssef1313 in [#4734](https://github.com/microsoft/testfx/pull/4734)
+* MSTEST0030: add codefix by @Evangelink in [#4769](https://github.com/microsoft/testfx/pull/4769)
+* Add Contains/DoesNotContain assert APIs by @Evangelink in [#4799](https://github.com/microsoft/testfx/pull/4799)
+* Add GitHubWorkItem by @Evangelink in [#4813](https://github.com/microsoft/testfx/pull/4813)
+* Disable MSTEST0026 by default by @Evangelink in [#4816](https://github.com/microsoft/testfx/pull/4816)
+* Add MSTest trace logs when using MTP by @Evangelink in [#4833](https://github.com/microsoft/testfx/pull/4833)
+* Fix some missing xml docs by @Evangelink in [#4841](https://github.com/microsoft/testfx/pull/4841)
```

Comment:
"Fixed" section?

Reply:
I was going to say housekeeping but as it is facing users, fixed is better.